pipeline/yaml_func.go

```diff
@@ -1,11 +1,19 @@
 package pipeline
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+)
 
 type YamlConfig struct {
 }
 
-func (yaml YamlConfig) Configure() error {
+func (yaml YamlConfig) GetComplexType(context *structContext) GoStructorValue {
 	fmt.Println("Level: Debug. Yaml configurator source run")
-	return nil
+	return NewGoStructorNoValue(context.Value.Interface(), errors.New("getcomplex type from yaml not implemented"))
+}
+
+func (yaml YamlConfig) GetBaseType(context *structContext) GoStructorValue {
+	fmt.Println("Level: Debug. Yaml configurator source run")
```

Comment:
use log not println

Reply:
no, that be enhancing by the way in a future. Right now it not should be like that